syndication/index.js

```diff
@@ -1,4 +1,4 @@
-import {$$, $} from '../utils';
+import {$$, $} from 'n-ui-foundations';
```

Comment:
document.querySelector and document.querySelectorAll helpers. Always been there, but not widely used.